mPower2/mPower2/AppDelegate.swift

```diff
@@ -170,6 +170,7 @@ class AppDelegate: SBAAppDelegate, RSDTaskViewControllerDelegate {
         // Otherwise, this will ask for permission to use location without any explanation on first launch
         // of the app.
         PassiveDisplacementCollector.shared.start()
+        PassiveGaitCollector.shared.start()
```

Comment:
The motion collection happens in the background so how does this manage the various permissioning scenarios?

Reply:
On start() it calls the RSDMotionRecorder's requestPermissions(on:, completion:) method, which checks authorization status, and if not authorized then request it. Since this happens the first time you sign in, it will either get authorized or denied at that time, while you've got the app open in front of you. As it's a passive collector, if it's denied the collector just won't run and we won't get their passive data, no harm done.